tests/service/ProjectAnalysisTests.fs

```diff
@@ -4325,3 +4325,63 @@ let ``Test project34 should report correct accessibility for System.Data.Listene
         |> Option.get
 
     listenerFuncEntity.Accessibility.IsPrivate |> shouldEqual true
+
+module Project35 = 
+    open System.IO
+
+    let fileName1 = Path.ChangeExtension(Path.GetTempFileName(), ".fs")
+    let base2 = Path.GetTempFileName()
+    let dllName = Path.ChangeExtension(base2, ".dll")
+    let projFileName = Path.ChangeExtension(base2, ".fsproj")
+    let fileSource1 = """
+type Test =
+    let curriedFunction (one:int) (two:float) (three:string) =
```

Comment:
let bindings in a type have no `ValReprInfo` either, thats why Ive always moaned about `CurrendParameterGroups`